menas/ui/service/EntityService.js

```diff
@@ -325,7 +325,22 @@ class DatasetService extends EntityService {
     }
   }
 
+  // if _properties with empty string exist, they will be replaced with undefs in place (backend expecting an empty option)
+  replaceBlankProperties(dataset) {
```

Comment:
I would do something similar on backend side as well. After all, the API can be called by anyone, any "enemy", not just this "friendly" UI.

Reply:
Could be. 
I wonder, however, if are general in the feature, are you absolutely certain we will not need empty string value, e.g. for enum as one value, so we can afford to do this?

Reply:
Done, an empty string for a dataset's value is converted to an _unset value_. Applies for creating & editing a dataset and for updating dataset's properties (test added to cover this, too.)